score_email.cwl

```diff
@@ -54,6 +59,8 @@ requirements:
           status = annots['prediction_file_status']
           del annots['prediction_file_status']
           if status == "SCORED":
+            for annot in args.private_annotaions:
+                del annots[annot]
             subject = "Submission to '%s' scored!" % evaluation.name
             message = ["Hello %s,\n\n" % syn.getUserProfile(userid)['userName'],
                        "Your submission (%s) is scored, below are your results:\n\n" % sub.name,
```

Comment:
Should we handle a case where 'no results' are returned.

Reply:
Do you mean the case where there is no result file sent to this tool from the scoring tool? That should only happen if that tool doesn't write a results.json, and in that case the tool would fail when it check outputs and the workflow would fail correct?

Reply:
In the scenario where we don't want any results written out. It should probably just say.
`Your submission () is scored`

Reply:
I'd do it like:

There's a check:
 if len(annots) == 0:
  message = "Your submission has been scored."
else: 
  message = <the current way of writing messages>